velox/exec/tests/TaskTest.cpp

```diff
@@ -615,6 +617,191 @@ TEST_F(TaskTest, singleThreadedCrossJoin) {
   }
 }
 
+struct ExternalBlocker {
```

Comment:
Can you make this a class and manipulate the fields through class methods so we can easily know how ExternalBlocker works by looking at the class itself? Thanks!